hostfactory/host_provider/src/cyclecloud_provider.py

```diff
@@ -961,7 +962,7 @@ def _deperecated_terminate_status(self, input_json):
         
         response["status"] = request_status
         
-        return self.json_writer(response)
+        return response
```

Comment:
Why didn't we write this to output_handler?

Reply:
Because we merge create_status and terminate_status in status call and at that point we write to output_handler